test/test_nestedtensor.py

```diff
@@ -1805,7 +1805,7 @@ def test_scaled_dot_product_attention(self, device, input_dim):
         def rand_tensor(*shape):
             return torch.randn(shape, device=device)
 
-        E = 10
+        E = 8
```

Comment:
This is to make it hit the fused kernels and cause the new guard to be triggered on multiple ragged sequence dims